crates/database/src/lib.rs

```diff
@@ -36,6 +37,7 @@ pub const ALL_TABLES: &[&str] = &[
     "presignature_events",
     "order_quotes",
     "solver_competitions",
+    "autopilot_auction",
```

Comment:
naming nit: why do we name the table prefixed with `autopilot_...`. I would slightly prefer `auctions` to be in line with how we name our other tables.

Reply:
Change the name to "auctions".